lp-platform/lp-platform-distributions/lp-platform-jetty-hsqldb/pom.xml

```diff
@@ -60,6 +60,18 @@
 			<type>jar</type>
 		</dependency>
 		<dependency>
+    		<groupId>org.xwiki.platform</groupId>
```

Comment:
Why do you need this 2 dependencies?  At least, I don't see how they're related to this PR.